couchbase/src/main/java/org/hibernate/ogm/datastore/couchbase/logging/impl/Log.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.couchbase.logging.impl;
+import static org.jboss.logging.Logger.Level.INFO;
+
+import org.hibernate.HibernateException;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * @author Stawicka Ewa
+ */
+@MessageLogger(projectCode = "OGM")
+public interface Log extends org.hibernate.ogm.util.impl.Log {
+	@LogMessage(level = INFO)
+	@Message(id = 1901, value = "Connecting to CouchBase at %s")
```

Comment:
You can start with 1701; Currently 1601 - 1700 is the upermost interval in use (by Cassandra).